src/app/profiles/profile-detail/profile-detail.component.html

```diff
@@ -31,23 +31,29 @@
             </mat-select>
           </mat-form-field>
 
-          <div fxFlex="50" style="margin-top:15px">
-            <mat-checkbox formControlName="generateRandomPassword">Generate Random AMT Password</mat-checkbox>
-          </div>
-          <mat-form-field fxFlex="100">
-            <input type="password" formControlName="amtPassword" matInput name="amtPassword" placeholder="AMT Password" required minlength=8 maxlength="32">
+          <mat-form-field fxFlex="75">
+            <input type="password" id="amtPassword" formControlName="amtPassword" matInput name="amtPassword" placeholder="AMT Password" required minlength=8 maxlength="32">
```

Comment:
![image](https://user-images.githubusercontent.com/22204562/129832586-191bdd7f-0612-4313-9fb9-61234afcdba1.png)
 
something got messed up :(

Reply:
Knew about this but I'm bad with front end design. I could use some help since I wrestled with the html and sass for a while and couldn't get it to look right